cephci/utils/configs.py

```diff
@@ -267,3 +267,22 @@ def get_reports(service):
         return _dict
     except KeyError:
         raise ConfigError(f"Insufficient config for '{service}'")
+
+
+def get_reporting_credentials(config, reporting_tool="database"):
+    get_configs(config)
+    if not CONFIG:
+        raise ConfigError("Configuration is not passed")
+    _dict = {}
+    try:
+        log.info(f"Loading credentials for reporting service '{reporting_tool}'")
+        reporting_creds = CONFIG["reports"][reporting_tool]
+        _dict["name"] = reporting_creds.get("db_name")
```

Comment:
`dict.get` returns `None` in case keyword is missing. `dict['key']` raises `KeyError` exception in case key is absent.